microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/project/MicroProfileConfigPropertyInformation.java

```diff
@@ -29,6 +29,7 @@
 	private final String propertyNameWithProfile;
 	private final String value;
 	private final String configFileName;
```

Comment:
If you have the URI, you can probably get derive the file name from the URI. eg. `Paths.get(sourceConfigFileUri).getName()`. The file name is just for display purposes.

Reply:
sourceConfigFileUri can be null, it's the reason why I did like that